Doc/library/unittest.mock.rst

```diff
@@ -1299,15 +1299,24 @@ patch.dict
     decorator. When used as a class decorator :func:`patch.dict` honours
     ``patch.TEST_PREFIX`` for choosing which methods to wrap.
 
+.. versionchanged:: 3.7
+
+    :func:`patch.dict` now returns the patched dictionary when used as a context
+    manager.
```

Comment:
I think this needs one more level of indentation, to be properly grouped with the doc for patch.dict.

Reply:
I've added that change and rebase. See the two outputs:

Without the extra indent:
![without indent](https://user-images.githubusercontent.com/3709683/55998071-a4c8c300-5cb4-11e9-9a68-0c02b49af403.PNG)

With the extra indent:
![with indent](https://user-images.githubusercontent.com/3709683/55998076-a85c4a00-5cb4-11e9-9277-7716e073440f.PNG)

If you think it looks better with the indent, all good, otherwise let me know so I can revert that.

Reply:
It’s a little weird that this is a block of doc about unittest.mock.patch.dict, but more doc follows out of the block (patch.dict can be used to add members etc).

I have to check the structure of the rest of the page before I can answer!

My concern is more about structure than looks; I recall a sphinx command that builds a changelog from the versionadded/versionchanged directives, with links back to the source pages; I’d just like these links to get to the correct section.